src/s3fs.cpp

```diff
@@ -737,7 +736,9 @@ int put_headers(const char* path, headers_t& meta, bool is_copy, bool use_st_siz
     //     get_object_attribute() returns error with initializing buf.
```

Comment:
This is the explanation of `ow_sse_flg` that was in the argument around v1.76.
It doesn't exist now, so we can delete it.

Reply:
Before v1.90, get_object_attribute() is called even if the file does not exist, and this is a comment at that time.
When calling from rename, get_object_attribute () was in error because the file did not exist.
In the current version, rename calls `use_st_size` as `false`, so this comment is incorrect.

If we write about this as `call use_st_size as false when the file does not exist(ex. rename object)`, that is correct.

Reply:
Done.